easybuild/easyconfigs/d/Devito/Devito-4.6.1-foss-2020a-Python-3.8.2.eb

```
@@ -5,7 +5,7 @@ version = '4.6.1'
 versionsuffix = '-Python-%(pyver)s'
 
 homepage = 'https://www.devitoproject.org'
-description = "Devito is a domain-specific Language (DSL) and code generation framework"
+description = "Devito is a domain-specific Language (DSL) and code generation framework for performing optimised Finite Difference (FD) computation from high-level symbolic problem definitions. Devito performs automated code generation and Just-In-time (JIT) compilation based on symbolic equations defined in SymPy to create and execute highly optimised Finite Difference stencil kernels on multiple computer platforms."
```

@georgebisbas This should be made multi-line (using `"""`), or it's going to trip our code style check (max line length in easyconfigs is 120 characters).

```suggestion
description = """Devito is a domain-specific Language (DSL) and code generation framework for
 performing optimised Finite Difference (FD) computation from high-level symbolic problem definitions.
 Devito performs automated code generation and Just-In-time (JIT) compilation based on symbolic
 equations defined in SymPy to create and execute highly optimised Finite Difference stencil kernels on
 multiple computer platforms."""